### ISSUE TITLE
Package request: jellyfin-media-player 1.9.1

### ISSUE THREAD

**Issue by Ember-ruby** [NixOS/nixpkgs#315573]
I think this previous version should still be packaged, as 1.10+ breaks compatibility with all older jellyfin instances

**Comment by bestlem** [comment]
What do you mean by previous version.

Also note that the current version seems to be 1.9.2 so an upgrade is needed.

**Comment by Ember-ruby** [comment]
> What do you mean by previous version.

1.9.x, as mentioned in the title

> Also note that the current version seems to be 1.9.2 so an upgrade is needed.

is it? their releases only show 1.9.1 https://github.com/jellyfin/jellyfin-media-player/releases

**Comment by bestlem** [comment]
I installed the mac app version direct from jellyfin.org  and got 1.9.2 and jellyfin-web in nixpkgs is 1.9.2 

The nixpkgs source includes
```
version = "10.9.1"; # ensure that jellyfin-web has matching version
```
You are looking at the media player whilst the source needed  here is the server https://github.com/jellyfin/jellyfin/releases

Github says 10.9.2 was released 2 weeks ago 
and 1.9.3 was released 3 days ago.

**Comment by Ember-ruby** [comment]
> Doesn't this mean that the media player is compatible with the current nixpkgs version of the server? 

yes, but the jellyfin server may be not hosted on nixos, or it may be hosted on nixos but using docker/podman

> Thus if you keep an old media player we also need an older 1.8.x server and web?

i don't know, i don't personally, but idk nixos' policies on this sort of thing

(i run it through podman on nixos, using a previous version, as the instance is for me alone, and i disagree with some of the changes in the latest version)

**Comment by eclairevoyant** [comment]
nixpkgs generally only packages one version of software at any given time. if you need a specific version, you should override the package locally (at minimum, `src`), or even copy the package expression locally and `callPackage` that.

**Comment by Ember-ruby** [comment]
> nixpkgs generally only packages one version of software at any given time. 

hm.
![Screenshot_08_2006301](https://github.com/NixOS/nixpkgs/assets/48266942/7a9ae3e0-b756-452b-895f-fb7633c5a1e8)

> if you need a specific version, you should override the package locally (at minimum, `src`), or even copy the package expression locally and `callPackage` that.

until there's an as easy as adding a regular package way, this is a shit workaround, and absolutely unreasonable to expect someone to do if there's a regression in a package but do not want to keep the entire system out of date.

**Comment by eclairevoyant** [comment]
> hm

this isn't some "gotcha", I said _generally_. Every version increases maintenance burden so we really avoid these sort of exceptions as much as possible.

Informing you to override is not a workaround, that's the design. You're asking nixpkgs to take on the maintenance burden of the override rather than handling it yourself.

**Comment by Ember-ruby** [comment]
> Informing you to override is not a workaround, that's the design. 

"until there's an as easy as adding a regular package way, this is a shit ~workaround~ design"

> What specifically is the issue with >1.9.1?

i've pretty clearly said what the issue is. but i suppose reading the issue would increase maintenance burden.

**Comment by eclairevoyant** [comment]
> i disagree with some of the changes in the latest version

I'm asking what you mean here. Cursing and sarcasm is not an effective way to get strangers to help you.

**Comment by Ember-ruby** [comment]
> > i disagree with some of the changes in the latest version
> 
> I'm asking what you mean here.

i do not think this is very relevant given the issue is that the latest client breaks compatibility with *all* previous servers (___unlike___ the android client)
i dislike the ui redesign, and the use of randomizing css classnames in it

> Cursing and sarcasm is not an effective way to get strangers to help you, particularly in a volunteer-driven project.

i have 'cursed' once/twice, and not even directed at someone, just as emphasis. lol.
